images/cilium/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ARG CILIUM_BUILDER_IMAGE=docker.io/cilium/cilium-builder-dev:879b19d3b784fdd0dda9f91310107c6fe209ffac
-ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:f31dddb7c313ef1ef1badc2992920151418f40d5
+ARG CILIUM_RUNTIME_IMAGE=docker.io/cilium/cilium-runtime-dev:0609ab6cc05c0dd78499c417c9ce3d0dc24c7f54
```

Comment:
You might need to update this again once https://github.com/cilium/cilium/pull/14798 is merged.

Reply:
The GitHub action is blocking on that at the moment anyway :/. `scripts/update-cilium-runtime-image.sh` provides a different tag, but I haven't figured out yet if I should just use the tag it retrieves or if I'm missing an update somewhere.

Reply:
Ok, waiting for your PR to be merged to rebase and run the CI. I'll un-draft in the meantime, as it is ready for reviews by original committers.

Reply:
@qmonnet The other backport PR was merged, you can rebase now.